src/tests/lib/qunit_config.js

```diff
@@ -1 +1,41 @@
-QUnit.config.testTimeout = 6400;
+/* globals badger:false */
+
+(function () {
+
+  let BACKUP = {};
+
+  QUnit.config.testTimeout = 6400;
+
+  // runs before any tests begin
+  QUnit.begin(() => {
+    // disable storage persistence
+    chrome.storage.local.set = () => {};
+
+    // set defaults
+    var settings = badger.storage.getBadgerStorageObject("settings_map");
+    for (let key in badger.defaultSettings) {
+      settings.setItem(key, badger.defaultSettings[key]);
```

Comment:
ah nice.

If privacy badger forms an opinion about example.com, could that mess up the unit tests? Could there be entries in the users action_map that effect the unit tests? Seems like something to trip on as well.